### ISSUE TITLE
XMaterial | Incorrect conversion of Porkchop

### ISSUE THREAD

**Issue by kingOf0** [CryptoMorin/XSeries#163]
`matchXMaterial(ItemStack)` converts Raw Porkchop to Cooked Porkchop.

**Code To Generate**
```
LOGGER.info(itemStack.type.name)
LOGGER.info(XMaterial.matchXMaterial(itemStack).name)
```
```
[20:30:16 INFO]: PORK
[20:30:16 INFO]: COOKED_PORKCHOP
```
       
Tested on **1.8.8** with XSeries **8.6.1**

Note: sorry for edit. I Miss clicked enter.

**Comment by CryptoMorin** [comment]
That code doesn't make sense. If `itemStack` variable is actually an `ItemStack` then it doesn't have a `type` field, I need to know how you constructed that variable.

**Comment by kingOf0** [comment]
Sorry it was in kotlin. In kotlin you can use ``getType()`` as a field like ``type``.
Heres a java example. (not written in an IDE)
```
ItemStack itemStack = new ItemStack(Material.PORK);
LOGGER.info(itemStack.getType().name);
LOGGER.info(XMaterial.matchXMaterial(itemStack).name);
```

**Comment by LOOHP** [comment]
> Sorry it was in kotlin. In kotlin you can use `getType()` as a field like `type`. Heres a java example. (not written in an IDE)
> 
> ```
> ItemStack itemStack = new ItemStack(Material.PORK);
> LOGGER.info(itemStack.getType().name);
> LOGGER.info(XMaterial.matchXMaterial(itemStack).name);
> ```

Should be `.name()`

**Comment by CryptoMorin** [comment]
Oh sorry, I didn't consider kotlin. Fixed in v8.6.2
